warplib/maths/external_input_checks.cairo

```diff
@@ -0,0 +1,265 @@
+# AUTO-GENERATED
```

Comment:
warplib files starting with AUTO-GENERATED should never be added to git. Generally the warplib gitignore handles this, but this is an old file that doesn't get generated anymore, and can be safely deleted